bigcrunch/webapp.py

```diff
@@ -39,13 +39,16 @@ def get_or_create(self):
             try:
                 response = yield from client.create_cluster(
                     ClusterIdentifier=cluster_identifier,
-                    NodeType='dc1.large',
+                    NodeType='dc2.large',
                     ClusterType='single-node',
                     VpcSecurityGroupIds=['sg-a8f55bcc'],
                     MasterUsername=username,
                     MasterUserPassword=password,
                     PubliclyAccessible=True,
                     ClusterParameterGroupName='default.redshift-1.0',
+                    IamRoles=[
+                        'arn:aws:iam::688441717003:role/SqlAlchemyRedshiftTestRole',
```

Comment:
I just created this role per the instructions in https://docs.aws.amazon.com/redshift/latest/gsg/rs-gsg-create-an-iam-role.html and should allow accessing the public sample `sales` dataset in S3.